src/commands/notify.yml

```
@@ -42,7 +42,12 @@ parameters:
       Select which channel in which to post to. Channel name or ID will work. You may include a comma separated list of channels if you wish to post to multiple channels at once. Set the "SLACK_DEFAULT_CHANNEL" environment variable for the default channel.
     type: string
     default: $SLACK_DEFAULT_CHANNEL
-
+  exit_on_fail:
+      description: |
+        Fail the pipeline if there has been an error posting to Slack.
+        This setting is useful mostly for initial setup to ensure the orb and Slack have been configured properly. Once established, it is recommended to disable this setting to ensure your pipelines are not inhibited by potential notification issues.
```

This could be a little more terse, and use present tense.

```suggestion
      description: |
        Fail on errors posting to Slack. Disable this to prevent Slack outages affecting your pipeline.
```

---

Again, excellent suggestion. Thank you for your input 👍. Applying all of these now.